### ISSUE TITLE
Waterheater:HeatPump:PumpedCondenser fan sizing error with more than one HPWH

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5560]
## Issue overview

User file fails to autosize the HPWH fan.

```
   ** Severe  ** SizeFan:  Fan:OnOff AUX HW LOOP 1 HP WATER HEATER SUPPLY FAN, Developer Error: Component sizing incomplete.
   **   ~~~   ** SizingString = Maximum Flow Rate [m3/s], SizingResult = -99999.0
```

There are two HPWH in this file.  From the eio output, it appears that the fan for the first HPWH is sized to match the flow of the second HPWH, and then the second fan gets left with no size.

```
 Component Sizing Information, WaterHeater:HeatPump:PumpedCondenser, HP WATER HEATER, Condenser water flow rate [m3/s], 3.70626E-002
 Component Sizing Information, WaterHeater:HeatPump:PumpedCondenser, HP WATER HEATER, Evaporator air flow rate [m3/s], 41.58910
 Component Sizing Information, WaterHeater:HeatPump:PumpedCondenser, AUX DHW LOOP 1 HP WATER HEATER, Condenser water flow rate [m3/s], 1.79480E-003
 Component Sizing Information, WaterHeater:HeatPump:PumpedCondenser, AUX DHW LOOP 1 HP WATER HEATER, Evaporator air flow rate [m3/s], 2.01400
 Component Sizing Information, Fan:OnOff, HP WATER HEATER SUPPLY FAN, Design Size Maximum Flow Rate [m3/s], 2.01400
```
### Workaround

Hard size the fan flow rates, using the HPWH evaporator flow rates reported in the eio output.
### Details

Some additional details for this issue (if relevant):
- Helpdesk ticket 11209
- pre-8.5.0 a070fda420
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] [Defect file added](https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/5000s/5560)
- [x] Ticket added to Pivotal for defect (development team task)
- [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
PR #5565 addresses this issue.

**Comment by Myoldmopar** [comment]
Closed via #5565
